src/components/common/input/AppInputWithButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { ForwardedRef, forwardRef } from 'react';
+import AppButton from '../button/AppButton';
+import styled from '@emotion/styled';
+import AppInput from './AppInput';
+
+const CategoriesInputSubContainer = styled.div`
```

Comment:
```suggestion
const CategoriesInputContainer = styled.div`
```

왜 sub container라고 지으셨나요??